ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -175,6 +175,7 @@ descriptor:
   next: "Something that occurs directly in time after the present or most recent one."
   "no": "Chemical formula and abbreviation for nitrogen monoxide, which is a colorless gas and a toxic air pollutant."
   no2: "Chemical formula and abbreviation for nitrogen dioxide."
+  nox: "Abbreviation for nitrogen oxides, which are a group of gases that are composed of nitrogen and oxygen."
```

Comment:
What is the difference between this and no/no2?

Reply:
Nox added because this device has a single sensor that measures both no and no2, this comes back as a single value, and cannot extract no and no2 from the nox value